test/integration/tf_cloudbuild_builder_simple/tf_cloudbuild_builder_simple_test.go

```diff
@@ -36,7 +36,7 @@ func TestTFCloudBuildBuilder(t *testing.T) {
 
 		projectID := bpt.GetStringOutput("project_id")
 		artifactRepo := bpt.GetStringOutput("artifact_repo")
-		artifactRepoDockerRegistry := fmt.Sprintf("us-docker.pkg.dev/%s/%s/terraform", projectID, artifactRepo)
+		artifactRepoDockerRegistry := fmt.Sprintf("us-central1-docker.pkg.dev/%s/%s/terraform", projectID, artifactRepo)
```

Comment:
I would recommend converting the (`us-central1`) location to a variable given it's repeated usage, and even consider plumbing it from a Terraform output.